doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -160,6 +160,10 @@ nRF Desktop
     You must use the alias to specify the QDEC instance used for scroll wheel, if your board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).
     You do not need to define the alias if your board supports only one QDEC instance, because in that case, the wheel module can rely on the ``qdec`` DT label provided by the board.
 
+* Updated:
+
+  * The :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` kconfig option value for the nRF52840 Gaming Mouse to better match the Fast Pair requirements.
```

Comment:
Maybe we could inform that it only affects configurations with Fast Pair support?

```suggestion
  * The :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option value in the nRF52840 Gaming Mouse configurations with Fast Pair support.
   The value was aligned with Fast Pair requirements.
```

Reply:
Addressed.